_tools/opengazettes.md

```diff
@@ -1,21 +1,38 @@
 ---
 layout: tool
+name: opengazettes
```

Comment:
How is it different? They're all like this (at least the ones in the `_tools` directory)